tests/runtests.py

```diff
@@ -436,14 +455,20 @@ def test_cmd_pattern_create(self):
         self.assertIn("aaaaaaaabaaaaaaacaaaaaaadaaaaaaa", res)
         return
 
+    @include_for_architectures(valid_architectures=["x86_64", "aarch64"])
     def test_cmd_pattern_search(self):
-        cmd = "pattern search $rbp"
+        if ARCH == "aarch64":
+            r = "$x30"
+        if ARCH == "x86_64":
+            r = "$rbp"
```

Comment:
Let's raise an exception if it escapes this.

Reply:
it won't enter the function if it's not in any specified arch, no need to raise